.github/workflows/pull-request.yaml

```diff
@@ -114,24 +120,21 @@ jobs:
   go-perf:
     name: Go Perf
     runs-on: ubuntu-22.04
+    needs: generate
     steps:
       - name: Check out code
         uses: actions/checkout@v3
 
+      - name: Download generated artifacts
+        uses: actions/download-artifact@v3
+        with:
+          name: generated
+
       - name: Benchmark Test Golang
         run: make ci-go-perf
         timeout-minutes: 30
-
-  go-quick-fuzz:
-    name: Go quick fuzz
-    runs-on: ubuntu-22.04
-    steps:
-      - name: Check out code
-        uses: actions/checkout@v3
-
-      - name: Run fuzz check (3m)
-        run: make ci-go-fuzz FUZZ_TIME=180s
-        timeout-minutes: 30
+        env:
+          DOCKER_RUNNING: 0
```

Comment:
Axing quick-fuzz is here likely fine. I've never observed it catching anything, and it's guaranteed to run for around 2-3 minutes, which can't be good for PR check latencies.